src/NRedisStack/Search/SearchResult.cs

```diff
@@ -12,6 +12,11 @@ public class SearchResult
         public long TotalResults { get; }
         public List<Document> Documents { get; }
 
+        /// <summary>
+        /// converts the documents to a list of json strings
+        /// </summary>
+        public IEnumerable<RedisValue> ToJson() => Documents.Select(x => x["json"]);
```

Comment:
What happens when you call ToJSON on a search , built off of HASH indexed data?

Reply:
after the last commit, empty list.